bootstrap/reset_db.py

```
@@ -0,0 +1,38 @@
+from dotenv import load_dotenv
+
+backend_dir = Path(__file__).resolve().parent.parent / "backend"
+sys.path.insert(0, str(backend_dir))
+load_dotenv(dotenv_path=backend_dir / ".env")
+
+from app.core.database import engine
+from sqlalchemy import text
+
+async def drop_all():
+    print("WARNING: This will delete ALL data in the database.")
+    confirm = input("Are you sure? Type 'YES' to continue: ")
```

Ruff (0.15.10)

[warning] Blocking call to `input()` in async context

(ASYNC250)